doc/source/whatsnew/v0.25.1.rst

```diff
@@ -103,7 +103,7 @@ MultiIndex
 I/O
 ^^^
 
--
+- Avoid calling ``S3File.s3`` when reading parquet as this was removed in s3fs version 0.3.0 (:issue:`27756`)
```

Comment:
```suggestion
- Avoid calling ``S3File.s3`` when reading parquet, as this was removed in s3fs version 0.3.0 (:issue:`27756`)
```

Reply:
Note the comma between "parquet" and "as"

Reply:
Thanks!